src/script_opt/CPP/RuntimeInit.h

```diff
@@ -20,6 +20,15 @@ typedef void (*CPP_init_func)();
 // Tracks the initialization hooks for different compilation runs.
 extern std::vector<CPP_init_func> CPP_init_funcs;
```

Comment:
Ideallly we'd stop using function pointers and start using `std::function` for similar things, but I realize there's a ton of those all over this code base. That might be a good big cleanup task for someone to pick up.

Reply:
Not an idiom on my radar, so I'll indeed leave it to others at some point in the future ...

Reply:
Yep, I'll write an issue for it so I don't forget about it.